modules/appeals_api/app/mailers/appeals_api/decision_review_mailer.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require 'appeals_api/decision_review_report'
+
+module AppealsApi
+  class DecisionReviewMailer < ApplicationMailer
+    RECIPIENTS = %w[
+      premal.shah@va.gov
+    ].freeze
+
+    def build(date_from:, date_to:)
+      @report = DecisionReviewReport.new(from: date_from, to: date_to)
+
+      template = File.read(path)
+      body = ERB.new(template).result(binding)
+
+      mail(
+        to: RECIPIENTS,
+        subject: 'Decision Review API report',
+        content_type: 'text/html',
+        body: body
+      )
+    end
+
+    private
+
+    def path
```

Comment:
It's odd this isn't done by default. Is this a result of the engine file structure?

Reply:
That's what I'm assuming... Just copy-pasting what I found with the other mailers. lol